XMLHttpRequest/open-url-redirected-worker-origin.htm

```diff
@@ -5,15 +5,14 @@
     <title>XMLHttpRequest: redirected worker scripts, origin and referrer</title>
     <script src="/resources/testharness.js"></script>
     <script src="/resources/testharnessreport.js"></script>
-    <link rel="help" href="https://xhr.spec.whatwg.org/#the-open()-method" data-tested-assertations="following::OL[1]/LI[3] following::OL[1]/LI[3]/ol[1]/li[1] following::OL[1]/LI[3]/ol[1]/li[2] following::OL[1]/LI[3]/ol[1]/li[3]" />
```

Comment:
Oh, this is cute, one of the previous unsuccessful attempts to ensure coverage of all musts that I've heard about.